mne/viz/_3d.py

```diff
@@ -1122,8 +1128,23 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
             opacity=0.75, glyph_height=0.25,
             glyph_center=(0., 0., 0.), glyph_resolution=20,
             backface_culling=True)
-    renderer.set_camera(azimuth=90, elevation=90,
-                        focalpoint=(0., 0., 0.), distance=0.6)
+    if view is None:
+        renderer.set_camera(azimuth=90, elevation=90,
+                            focalpoint=(0., 0., 0.), distance=0.6)
+    elif isinstance(view, dict):
+        renderer.set_camera(azimuth=view.get('azimuth', 90),
+                            elevation=view.get('elevation', 90),
+                            focalpoint=view.get('focalpoint',
+                                                (0., 0., 0.)),
+                            distance=view.get('distance', 0.6))
```

Comment:
I like the 	`view.get('key', or_default)`. But If you do this, I would do something like:

```py
_check_value(view, (dict, None))
view = dict() if view is None else view
render.set_cammera(azimuth=view.get('azimuth', 90),
                            elevation=view.get('elevation', 90),
                            focalpoint=view.get('focalpoint',
                                                (0., 0., 0.)),
                            distance=view.get('distance', 0.6)))
```

and I even think that if you put the defaults in `set_cammera` and allow a `*kwargs` something like this should work:

```py
_check_value(view, (dict, None))
view = dict() if view is None else view
render.set_cammera(**view)
```

Reply:
yes it works!! 

```py
def set_camera(azimuth=90, elevation=90, focalpoint=(0., 0., 0.),
               distance=0.6, **dump_parameters):
    print('azimuth :', azimuth,
          ', elevation :', elevation,
          ', focalpoint :', focalpoint,
          ', distance :', distance)


set_camera(**dict())
set_camera(**dict(azimuth='az', focalpoint='focal'))
set_camera(**dict(azimuth='az', not_here='not_here'))
```

Reply:
I like it, this is very compact and it's straightforward to pass the parameters that way.

Reply:
its a way of saying: I have this dict of paramameters use what you can.

Notice that you need `**dump_parameters` otherwise the 3rd call would give you an error. I would log/warn/error if `dump_parameters` is not empty dict

Reply:
or maybe is a good idea to not have `dump_parameters` like this if you pass something that it's not supposed to be passed it would crash. Up to you